docs/design/admission_control_limit_range.md

```diff
@@ -209,6 +210,25 @@ Across all containers in pod, the following must hold true
 | Max | Limit (required) <= Max |
 | LimitRequestRatio | LimitRequestRatio <= ( Limit (required, non-zero) / Request (non-zero) ) |
 
+**Type: PersistentVolumeClaim**
+
+Supported Resources:
+
+1. storage
+
+Supported Constraints:
+
+Across all claims in a namespace, the following must hold true:
+
+| Constraint | Behavior |
+| ---------- | -------- |
+| Min | Min >= Request (required) |
+| Max | Max <= Request (required) |
+
+Supported Defaults: None. Storage is a required field in `PersistentVolumeClaim`, so defaults are not applied at this time.
```

Comment:
Many fields that are required are assigned server side if omitted, I had thought you needed that use case.  

It's fine not doing it now, I was just surprised.